gulpfile.js

```diff
@@ -1,4 +1,14 @@
-const moduleError = /Error: Cannot find module '([a-zA-Z0-9+_-]+)'/g;
+// The bot will have other problems if they're using Node v6<, so the only version
+// we probably need to worry about is v6.
+if (process.version.startsWith('v6') || typeof Object.values === 'undefined') {
```

Comment:
`Object.values` is the backup check. It's only available in Node v7 and up. In fact, we could have *just* that if you wanted.